config/robots.rb

```diff
@@ -3727,4 +3727,252 @@
+          purpose: 'LCMT Lysate',
+          states: ['passed'],
+          label: 'Bed 4'
+        },
+        car('1,4').barcode => {
+          purpose: 'LCMT DNA Frag',
+          states: ['pending'],
+          label: 'Carousel 1,4',
+          parent: bed(4).barcode
+        },
+        car('2,4').barcode => {
+          purpose: 'LCMT DNA End Prep',
+          states: ['pending'],
+          label: 'Carousel 2,4',
+          parent: car('1,4').barcode
+        },
+        car('3,5').barcode => {
+          purpose: 'LCMT DNA Adp Lig',
+          states: ['pending'],
+          label: 'Carousel 3,5',
+          parent: car('2,4').barcode
+        },
+        car('4,4').barcode => {
+          purpose: 'LCMT EM TET2 Ox',
+          states: ['pending'],
+          label: 'Carousel 4,4',
+          parent: car('3,5').barcode
+        },
+        car('4,3').barcode => {
+          purpose: 'LCMT EM TET2 Stop',
+          states: ['pending'],
+          label: 'Carousel 4,3',
+          parent: car('4,4').barcode
+        }
+      }
+    )
+
+    # LCM Triomics WGS and EMSeq bed verification
+    # Bravo LCMT DNA Frag Verification
+    custom_robot(
+      'bravo-lcmt-dna-frag-verification',
+      name: 'Bravo LCMT DNA Frag Verification',
+      require_robot: false,
+      beds: {
+        bed(5).barcode => {
+          purpose: 'LCMT DNA Frag',
+          states: ['pending'],
+          label: 'Bed 5',
+          target_state: 'passed'
+        }
+      }
+    )
+
+    # LCM Triomics WGS and EMSeq bed verification
+    # Bravo LCMT DNA End Prep Verification
+    custom_robot(
+      'bravo-lcmt-dna-end-prep-verification',
+      name: 'Bravo LCMT DNA End Prep Verification',
+      require_robot: false,
+      beds: {
+        bed(5).barcode => {
+          purpose: 'LCMT DNA End Prep',
+          states: ['pending'],
+          label: 'Bed 5',
+          target_state: 'passed'
+        }
+      }
+    )
+
+    # LCM Triomics WGS and EMSeq bed verification
+    # Bravo LCMT DNA Adp Lig to LCMT DNA Lib PCR
+    custom_robot(
+      'bravo-lcmt-dna-adp-lig-to-lcmt-dna-lib-pcr',
+      name: 'Bravo LCMT DNA Adp Lig => LCMT DNA Lib PCR',
+      require_robot: false,
+      beds: {
+        bed(6).barcode => {
+          purpose: 'LCMT DNA Adp Lig',
+          states: ['passed'],
+          label: 'Bed 6'
+        },
+        bed(7).barcode => {
+          purpose: 'LCMT DNA Lib PCR',
+          states: ['pending'],
+          label: 'Bed 7',
+          parent: bed(6).barcode,
+          target_state: 'passed'
+        }
+      }
+    )
+
+    # LCM Triomics WGS and EMSeq bed verification
+    # Bravo LCMT EM TET2 Ox Verification
+    custom_robot(
+      'bravo-lcmt-em-tet2-ox-verification',
+      name: 'Bravo LCMT EM TET2 Ox Verification',
+      require_robot: false,
+      beds: {
+        bed(5).barcode => {
+          purpose: 'LCMT EM TET2 Ox',
+          states: ['pending'],
+          label: 'Bed 5',
+          target_state: 'passed'
```

Comment:
again, not passed state for a verification (no transfer)

Reply:
there is a bed 5 check for each after the physical transfer happens.